### PR TITLE
Update README (#269)

### LINKED ISSUE #269: Update README to reflect new scraper changes
The current [README.md](https://github.com/openzim/youtube/blob/main/README.md) needs to be updated to reflect the new scraper changes. Some of the issues I noticed are:

1.  https://github.com/openzim/youtube/blob/1ba45c384cd8296a550fce17fb5bd509dd7748c8/README.md?plain=1#L16-L17 We are no longer creating static HTML files, so this could be replaced by mentioning the new Vue.js UI.

2. https://github.com/openzim/youtube/blob/1ba45c384cd8296a550fce17fb5bd509dd7748c8/README.md?plain=1#L27 All Javascript dependencies are handled through Yarn, so this line could be replaced with something mentioning Yarn.

3. https://github.com/openzim/youtube/blob/1ba45c384cd8296a550fce17fb5bd509dd7748c8/README.md?plain=1#L44 The developer setup wiki link is broken. Probably should be replaced with https://github.com/openzim/_python-bootstrap/blob/main/docs/Developer-Setup.md.

4. https://github.com/openzim/youtube/blob/1ba45c384cd8296a550fce17fb5bd509dd7748c8/README.md?plain=1#L86 `--only_test_branding` CLI argument doesn't exist anymore. This line should be removed.

### PR DESCRIPTION
**Pull request by dan-niles**
Close #269

Update README to reflect the new scraper changes implemented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,6 @@ youtube2zim --api-key "<your-api-key>" --type user --id "Vsauce" --name "vsauce"
 
 ## Notes
 
-* Your API_KEY is subject to usage quotas (10,000 requests/day) so use `--only_test_branding` when adjusting parameters and branding to not *waste your quota*.
```
Comment (benoit74) [inline README.md:86]: Even if parameter is gone, I would keep the warning with something like "Your API_KEY is subject to usage quotas (10,000 requests/day by default) so be careful to not *waste your quota*, especially when scraping or testing big channels.

File: README.md [hunk 2]
```diff
@@ -13,18 +13,18 @@ from a Youtube Channel/Username or one-or-more Playlists.
...
 
 `youtube2zim` adheres to openZIM's [Contribution Guidelines](https://github.com/openzim/overview/wiki/Contributing).
 
-`youtube2zim` has implemented openZIM's [Python bootstrap, conventions and policies](https://github.com/openzim/_python-bootstrap/docs/Policy.md) **v1.0.2**.
+`youtube2zim` has implemented openZIM's [Python bootstrap, conventions and policies](https://github.com/openzim/_python-bootstrap/blob/main/docs/Policy.md) **v1.0.2**.
 
 Requirements
 ------------
 
 * [`ffmpeg`](https://ffmpeg.org/) for video transcoding (only used with `--lower-quality`).
```
Comment (benoit74) [inline README.md:26]: ffmpeg is now always used, the `--low-quality` only switches between high and low presets. We should remove the "(only used with ...)" statement.

File: README.md [hunk 3]
```diff
@@ -13,18 +13,18 @@ from a Youtube Channel/Username or one-or-more Playlists.
 
 It downloads the video (`webm` or `mp4` extension – optionally
 recompress them in lower-quality, smaller size), the thumbnails, the
-subtitles and the authors' profile pictures ; then, it create a static
-HTML files folder of it before creating a ZIM off of it.
+subtitles and the authors' profile pictures ; then produces JSON files
+containing content for the channel, playlists, and videos, which are then utilized by a Vue.js UI.
```
Comment (benoit74) [inline README.md:18]: We need an additional statement explaining a bit the existence of the Vue.JS part. Something like "UI is a Vue.JS application which needs to be compiled as a static website with Vite and is them embedded inside the ZIM."

File: README.md [hunk 4]
```diff
@@ -83,7 +83,6 @@ youtube2zim --api-key "<your-api-key>" --type user --id "Vsauce" --name "vsauce"
 
 ## Notes
 
-* Your API_KEY is subject to usage quotas (10,000 requests/day) so use `--only_test_branding` when adjusting parameters and branding to not *waste your quota*.
 * If you encounter issues reading ZIM files created using this scraper, please take a look at the [Compatibility Matrix](https://github.com/openzim/youtube/wiki/Compatibility) before opening a ticket.
```
Comment (benoit74) [inline README.md:86]: We should remove this statement, compatibility matrix, is now OK for all readers (compatibility matrix should be updated btw)

**Review by benoit74 — CHANGES_REQUESTED** [review]
Small additions now that we are revisiting the README ^^

File: README.md (hunk 4 above)
Comment (dan-niles) [reply README.md]: Removed in [7f7c5bc](https://github.com/openzim/youtube/pull/271/commits/7f7c5bc44c001b2ec089372907973526c45f7ee4). 
Created #273 to track updating the compatibility matrix.

File: README.md (hunk 3 above)
Comment (dan-niles) [reply README.md:17]: Updated in [7f7c5bc](https://github.com/openzim/youtube/pull/271/commits/7f7c5bc44c001b2ec089372907973526c45f7ee4).

File: README.md (hunk 2 above)
Comment (dan-niles) [reply README.md]: Removed in [7f7c5bc](https://github.com/openzim/youtube/pull/271/commits/7f7c5bc44c001b2ec089372907973526c45f7ee4).

File: README.md (hunk 1 above)
Comment (dan-niles) [reply README.md:86]: Updated in [7f7c5bc](https://github.com/openzim/youtube/pull/271/commits/7f7c5bc44c001b2ec089372907973526c45f7ee4).

**Review by benoit74 — APPROVED** [review]
LGTM

**Pull request merged**